tasks/fix-cat2.yml

```diff
@@ -1511,12 +1552,15 @@
       - RHEL-07-040740
 
 - name: "MEDIUM | RHEL-07-040810 | PATCH | The system must use a local firewall."
```

Comment:
Actually, this is a duplicate of RHEL-07-040290. Slightly different wording on the requirement but the fix action for both is identical. We can remove these tasks and just leave a comment in line with the finding ID stating it's a duplicate.

Hooray for working from the draft STIG.